shell/browser/api/electron_api_app.cc

```diff
@@ -677,9 +677,6 @@ void App::OnFinishLaunching(const base::DictionaryValue& launch_info) {
 
 void App::OnPreMainMessageLoopRun() {
   content::BrowserChildProcessObserver::Add(this);
-  if (process_singleton_) {
-    process_singleton_->OnBrowserReady();
-  }
```

Comment:
A little surprised that removing this is correct..?

Reply:
Took a look again. I'll add that part back, and also add more to the `process_singleton_posix.cc` patch to handle that case (right now it's just a DCHECK still https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/process_singleton_posix.cc;l=1048?q=process_singleton_posix&ss=chromium)